pkg/auth/iguazio/v1/auth.go

```diff
@@ -58,7 +57,7 @@ func (a *Auth) Authenticate(request *http.Request, options *authpkg.Options) (au
 	}
 
 	if cookie == "" && authorization == "" {
-		return nil, nuclio.NewErrForbidden("Authentication headers are missing")
+		return nil, nuclio.NewErrUnauthorized("Authentication headers are missing")
```

Comment:
Need to test it on IGZ3 and see that it doesn't break anything (it shouldn't)

Reply:
I will revert that